test/e2e/framework/util.go

```diff
@@ -296,31 +322,44 @@ func waitForServiceAccountInNamespace(c clientset.Interface, ns, serviceAccountN
 	}
 	ctx, cancel := watchtools.ContextWithOptionalTimeout(context.Background(), timeout)
 	defer cancel()
-	_, err := watchtools.UntilWithSync(ctx, lw, &v1.ServiceAccount{}, nil, serviceAccountHasSecrets)
```

Comment:
now that admission no longer uses this, we should drop this wait entirely

Reply:
I need to check whether anyone depends on the token or has an explicit wait then.

Reply:
Looks like no one explicitly tries to read the default token in core e2e.